doc/user-guides/dns/orphan-dns-records.md

```diff
@@ -24,33 +24,37 @@ On cluster 2 the DNS Operator managing the existing DNSRecord in that cluster ha
 In prometheus alerts, it spots that the number of owners does not correlate to the number of DNSRecord resources and triggers an alert. 
 To remedy this rather than going to the DNS provider directly and trying to figure out which records to remove, you can instead follow the steps below.
 
-1) Get the owner id of the DNSRecord on cluster 2 for the shared host 
+Get the owner id of the DNSRecord on cluster 2 for the shared host 
 
-```
+```sh
 kubectl get dnsrecord somerecord -n my-gateway-ns -o=jsonpath='{.status.ownerID}'
 ```
 
-2) get all the owner ids
+Get all the owner ids
 
-```
+```sh
 kubectl get dnsrecord.kuadrant.io somerecord -n my-gateway-ns -o=jsonpath='{.status.domainOwners}'
 
-## output
-["26aacm1z","49qn0wp7"]
+# output
+# ["26aacm1z","49qn0wp7"]
 ```
 
-3) create a placeholder DNSRecord with none active ownerID
+Create a placeholder DNSRecord with none active ownerID
 
 
-for each owner id returned that isn't the owner id of the record we got earlier that we want to remove records for, we need to create a dnsrecord resource and delete it. This will trigger the running operator in this cluster to clean up those records.
+For each owner id returned that isn't the owner id of the record that we want to remove records for, we need to create a dnsrecord resource and delete it. This will trigger the running operator in this cluster to clean up those records.
```

Comment:
```suggestion
For each owner ID returned that isn't the owner ID of the record that we want to remove records for, we need to create a DNSRecord resource and delete it. This will trigger the running operator in this cluster to clean up those records.
```